authn/postgres/setup_test.go

```diff
@@ -18,6 +18,7 @@ import (
 )
 
 const wrong string = "wrong-value"
+const expInterval = 60 // in seconds
```

Comment:
Just a curiosity - how did we get to this interval and is it enough for test to be run in all cases?

Reply:
This is a good question because it depends on each case, on the other hand, you can set the value with a large margin. I just had to start with a certain value and it seemed to me that a minute would be fine. Maybe you have a suggestion?